docs/networking/harvester-network.md

```diff
@@ -8,7 +8,7 @@ keywords:
 ---
 # Network
 
-Harvester provides three types of virtual network devices for virtual machines (VMs), including:
+Harvester provides three types of virtual network for virtual machines (VMs), including:
```

Comment:
```suggestion
Harvester provides three types of virtual networks for virtual machines (VMs), including:
```

Reply:
I was translating the document and found that in the sentence `2. Click the **Create Network Config** button on the left of the cluster network to create a new network configuration.` in the cluster network doc, the word `left` should be `right` according to the screenshot. 
If that's true, is it ok to help fix it in this PR as well? Thanks!! @yaocw2020